### PR TITLE
added instructions for AWS

### PR DESCRIPTION
**Pull request by arun-gupta**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/dev-machine-setup.md [hunk 1]
```diff
@@ -21,7 +21,28 @@ outlined in the [Getting Started](getting-started.md) instructions).
 
 ### AWS
 
-`[TODO]`
+Firecracker development environment on AWS can be setup using bare metal instances. Follow these steps to create a bare metal instance.
+
+1. If you don't already have an AWS account, create one: https://portal.aws.amazon.com/billing/signup.
+1. Install AWS CLI: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html
```
Comment (andreeaflorescu) [inline docs/dev-machine-setup.md]: This step is not mandatory because you can start the instance from the AWS console. Also, it seems like you are not using the AWS cli anyway.

File: docs/dev-machine-setup.md [hunk 2]
```diff
@@ -21,7 +21,28 @@ outlined in the [Getting Started](getting-started.md) instructions).
...
+1. In `Step 3`, click on `Next: Add Storage`.
+1. In `Step 4`, click on `Next: Add Tags`.
+1. In `Step 5`, click on `Next: Configure Security Group`.
+1. In `Step 6`, take the default security group. This opens up port 22 and is needed so that you can ssh into the machine later. Click on `Review and Launch`.
+1. Verify the details and click on `Launch`. If you do not have an existing key pair, then you can select `Create a new key pair` to create a key pair. This is needed so that you can use it later to ssh into the machine.
+1. Click on the instance id in the green box. Copy `Public DNS` from the `Description` tab of the selected instance.
+1. Login to the newly created instance:
+
+   ```
+   ssh -i <ssh-key> ubunut@<public-ip>
```
Comment (dianpopa) [inline docs/dev-machine-setup.md]:
```suggestion
   ssh -i <ssh-key> ubuntu@<public-ip>
```

File: docs/dev-machine-setup.md [hunk 3]
```diff
@@ -21,7 +21,28 @@ outlined in the [Getting Started](getting-started.md) instructions).
...
+   ```
+
+  Now you can continue with the Firecracker [Getting Started](getting-started.md)
+  instructions to install and configure Firecracker in the new VM.
```
Comment (dianpopa) [inline docs/dev-machine-setup.md]: I would rephrase this sentence (i.e `...instructions to install and configure Firecracker in the new VM`) since you are not installing Firecracker in a VM you are using Firecracker to spawn a microVM.

File: docs/dev-machine-setup.md [hunk 4]
```diff
@@ -21,7 +21,27 @@ outlined in the [Getting Started](getting-started.md) instructions).
 
 ### AWS
 
-`[TODO]`
+Firecracker development environment on AWS can be setup using bare metal instances. Follow these steps to create a bare metal instance.
```
Comment (andreeaflorescu) [inline docs/dev-machine-setup.md]: Can you make you all the lines maximum 80 characters long?

File: docs/dev-machine-setup.md [hunk 5]
```diff
@@ -21,7 +21,27 @@ outlined in the [Getting Started](getting-started.md) instructions).
...
+
+1. If you don't already have an AWS account, create one: https://portal.aws.amazon.com/billing/signup.
```
Comment (andreeaflorescu) [inline docs/dev-machine-setup.md]: nit: do not use bare URL in markdown. One suggestion would be to re-write it as:
```
... create on using the [AWS portal](https://portal.aws.amazon.com/billing/signup).
```
or whatever else sounds better.

File: docs/dev-machine-setup.md [hunk 6]
```diff
@@ -21,7 +21,27 @@ outlined in the [Getting Started](getting-started.md) instructions).
...
+1. Login to AWS console: https://console.aws.amazon.com/console/home?region=us-east-1. Bare metal instances are only supported in `US East (N. Virginia)` region at this time. This region is preselected for you in the Console.
```
Comment (andreeaflorescu) [inline docs/dev-machine-setup.md]: Bare URL, can you re-write it as:
```
Login to the [AWS Console](https://console.aws.amazon.com/console/home?region=us-east-1).
```
